### ISSUE TITLE
3.0.0: constant names prefixed with dollar sign

### ISSUE THREAD

**Issue by jrfnl** [phpDocumentor/phpDocumentor#2428]
**Given the following piece of code in file `src/issue-2428.php`:**
``` php
<?php

const GLOBAL_CONST = 'foo';

class Issue2428 {
    const CLASS_CONST = 'bar';
}
```

<details>
  <summary><b>with the following config:</b></summary>

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<phpdocumentor
    configVersion="3"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns="https://www.phpdoc.org"
    xsi:noNamespaceSchemaLocation="https://docs.phpdoc.org/latest/phpdoc.xsd"
>
    <title>Demo</title>
    <version number="latest">
        <api format="php">
            <source dsn=".">
                <path>src</path>
            </source>
        </api>
    </version>
    <template name="default"/>
</phpdocumentor>
```
</details>

## Expected behavior
I'd expect to see the constant name displayed as declared...

## Actual behavior
... instead of prefixed with a dollar `$` sign.

>![issue-2428-global](https://user-images.githubusercontent.com/663378/84693716-37639280-af48-11ea-8e7f-fbebb213e0c7.png)

>![issue-2428-class](https://user-images.githubusercontent.com/663378/84693715-36cafc00-af48-11ea-959a-782658cff138.png)

## As found in project

* [Direct link to the source code where the issue was found](https://github.com/PHPCSStandards/PHPCSUtils/blob/7a54196a2c079dae38d2bd5b5eb605f514d9c3e6/PHPCSUtils/Utils/Numbers.php#L38-L45)
* [Direct link to the HTML source of the generated docs](https://github.com/PHPCSStandards/PHPCSUtils/blob/develop/docs/phpdoc/classes/PHPCSUtils-Utils-Numbers.html#L235)
* [Output as seen online](https://phpcsutils.com/phpdoc/classes/PHPCSUtils-Utils-Numbers.html#constant_REGEX_DECIMAL_INT)

## Steps to reproduce the problem

1. Issue can be reproduced with the above code sample and config or via the linked project.
2. Run `php -f /path/to/phpDocumentor/bin/phpdoc`

## Your environment

* Version used: phpDocumentor v3.0.x-dev@d15f2d6
* Install method: git clone
* PHP version: 7.4.6
* Operating system and version: Windows 7
* Link to your project: https://github.com/PHPCSStandards/PHPCSUtils/
* Link to output as committed: https://github.com/PHPCSStandards/PHPCSUtils/tree/develop/docs/phpdoc
* Link to output online: https://phpcsutils.com/phpdoc/

**Comment by mvriel** [comment]
A good old fashioned copy/paste mistake, fixed in commit cfc333b8bf175ed1297a5308abb9583c7742d418

**Comment by jrfnl** [comment]
Don't you just love those :sunglasses:  Tested & fix confirmed. Thanks for the quick turn-around @mvriel !
